Makefile

```diff
@@ -1,5 +1,9 @@
 
 prefix ?= /usr/local
+# fall back to $prefix/etc (like autoconf); see discussion on #222
+#sysconfdir ?= $(prefix)/etc)
```

Comment:
I don't think this provides anything useful, once you need to override `sysconfdir` a fallback atlernative doesn't really matter.